src/components/Section/History/Employment/EmploymentItem.jsx

```diff
@@ -516,6 +517,7 @@ export default class EmploymentItem extends ValidationElement {
               address: this.props.ReferenceAlternateAddress,
               belongingTo: 'ReferenceAlternateAddress',
               country: this.props.ReferenceAddress.country,
+              militaryAddressLabel: i18n.t(`${prefix}.heading.militaryAddress`),
```

Comment:
@christopherhuii Here's where those 2 text strings get inserted. This section has a whole bunch of text that is similar, but different between self/unemployment so they group them under common subpath and use `prefix` to distinguish, which is set to self/unemployment specific parent paths.